<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserAgent</class>
 <widget class="QWidget" name="UserAgent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>552</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QGroupBox" name="customUABox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Existing identifications</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="KMessageWidget" name="invalidTemplateNameWidget"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QTreeWidget" name="templates">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="showDropIndicator" stdset="0">
             <bool>false</bool>
            </property>
            <property name="rootIsDecorated">
             <bool>false</bool>
            </property>
            <property name="uniformRowHeights">
             <bool>true</bool>
            </property>
            <property name="itemsExpandable">
             <bool>false</bool>
            </property>
            <property name="allColumnsShowFocus">
             <bool>true</bool>
            </property>
            <property name="columnCount">
             <number>2</number>
            </property>
            <column>
             <property name="text">
              <string>Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Content</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="newTemplateBtn">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>&amp;New...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="editTemplateBtn">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string/>
            </property>
            <property name="text">
             <string>&amp;Edit...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="renameTemplateBtn">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string/>
            </property>
            <property name="text">
             <string>&amp;Rename...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="duplicateTemplateBtn">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string/>
            </property>
            <property name="text">
             <string>Du&amp;plicate...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteTemplateBtn">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string/>
            </property>
            <property name="text">
             <string>&amp;Delete</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="useTemplateBtn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Choose &amp;selected identification</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="useDefaultUA">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If checked, disable choosing a custom identification and use the default one&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Use &amp;default identification</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>&amp;Custom identification</string>
       </property>
       <property name="buddy">
        <cstring>userAgentString</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="userAgentString">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The browser identification to use when the default identification is disabled.&lt;/p&gt;&lt;p&gt;You can enter it manually or start from one in the list below and change it according to your needs.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KMessageWidget</class>
   <extends>QFrame</extends>
   <header>kmessagewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
